src/components/navbar/index.tsx

```diff
@@ -66,20 +75,34 @@ const Navbar: React.FC = (): JSX.Element => {
     setComponentViewSave("TutorialSelector");
   };
 
+  const openTutorialEditor = (): void => {
+    setComponentView("TutorialEditor");
+    setComponentViewSave("TutorialEditor");
+  };
+
   return (
     <NavWrapper>
-      <NavLogo to="/" onClick={openTutorialComponent}>
+      <NavLogo
+        to="/"
+        onClick={openTutorialComponent}
+        isSelected={componentView === "TutorialComponent"}
```

Comment:
Might have to make these into their own separate components and pass isselected as a prop to avoid console error.